main.go

```diff
@@ -14,6 +14,10 @@ func main() {
 		return
 	}
 
+	if string(data) == "" {
+		return
+	}
+
```

Comment:
That's a special case where the file is empty. I complained about the more general case where there's just no final \n:

```
➜  trainee-sort git:(feature/implement-sort-from-scratch-1) sort <<<'LOLCAT'
LOLCAT
➜  trainee-sort git:(feature/implement-sort-from-scratch-1) go run . <<<'LOLCAT'

LOLCAT
➜  trainee-sort git:(feature/implement-sort-from-scratch-1)
```

Reply:
So i don't want to print empty lines that are e.g. at the beginning at the end or somewhere in the file? So i just take out those and print afterwards. Should i undo these changes?

Reply:
No, it's just about the final \n:

```
➜  trainee-sort git:(feature/implement-sort-from-scratch-1) sort <<<'a

b'

a
b
➜  trainee-sort git:(feature/implement-sort-from-scratch-1) go run . <<<'a

b'


a
b
➜  trainee-sort git:(feature/implement-sort-from-scratch-1)
```